warehouse/query-core/src/main/java/datawave/query/planner/DefaultQueryPlanner.java

```diff
@@ -1245,6 +1229,18 @@ protected ASTJexlScript processTree(final ASTJexlScript originalQueryTree, Shard
                 }
                 innerStopwatch.stop();
                 
+                List<String> debugOutput = null;
+                if (log.isDebugEnabled()) {
+                    debugOutput = new ArrayList<>(32);
+                }
+                
+                /**
+                 * If the query is executable we can check and expand composite terms if the feature is enabled and following range and regex expansion.
+                 */
+                if (ExecutableDeterminationVisitor.isExecutable(queryTree, config, indexedFields, indexOnlyFields, nonEventFields, debugOutput, metadataHelper)) {
+                    // queryTree = checkAndExpandCompositeTerms(config, queryTree, timers, "Post-regex expansion");
+                }
```

Comment:
Is there a side effect to the isExecutable call, or should the whole block be removed?

Reply:
Derp I just saw this when I was going through this a second time. . I'm also going to remove the log statements above. they don't add a lot of value where they are at. Thanks!

Reply:
I take that back. I'm going to keep one.